packages/ia-components/sandbox/details/AnchorDownload.jsx

```diff
@@ -78,36 +78,39 @@ function downloadViaAnchor(el, source, options, cb) {
 }
 
 function reachable({ disconnected, source, identifier, filename }) {
+  // Note that multi element arrays arent reachable if disconnected because they need "compress" which we dont currently support on dweb-mirror
   return (!disconnected
     || (source
-      ? (Array.isArray(source) ? ((source.length === 1) && source[0].downloaded) : source.downloaded )
+      ? (Array.isArray(source) ? source.every(s => s.downloaded) : source.downloaded )
```

Comment:
is this just a pure refactor or are you looking for an actual value? `every` seems to returns a boolean, and not the value prior to this change.
`https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every`

Reply:
Yes - I want a boolean - (the ARCHIVEFILE.downloaded field is a boolean), so both cases return a boolean - old code was only checking the first file (for some strange reason i cant figure out), and it needs to check all of them.